From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Surface Duo Builder <bariccatti@gmail.com>
Date: Sun, 19 Apr 2026 00:00:00 +0000
Subject: [PATCH 1/1] EGL: Suppress unused variable warning in Loader.cpp

LineageOS fork sets failToLoadFromDriverSuffixProperty but never reads it.
Silence -Wunused-but-set-variable to fix -Werror build failure.
---
 opengl/libs/EGL/Loader.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/opengl/libs/EGL/Loader.cpp b/opengl/libs/EGL/Loader.cpp
index 0000000..1111111 100644
--- a/opengl/libs/EGL/Loader.cpp
+++ b/opengl/libs/EGL/Loader.cpp
@@ -310,6 +310,7 @@
             // Abort regardless of whether subsequent properties are set, the value must be set
             // correctly with the first property that has a value.
             break;
+    (void)failToLoadFromDriverSuffixProperty;
         }
     }
 
-- 
2.34.1
